vpr/src/place/move_generator.h

```diff
@@ -54,8 +59,9 @@ class MoveGenerator {
      *  @param rlim: maximum distance a block can move in x or y direction, in the compressed grid space
      *  @param placer_opts: all the placer options
      *  @param criticalities: the placer criticalities, useful for timing directed moves
+     *  @param blk_type: function proposes a move with given block type if specified.  
```

Comment:
If -1, this function will choose the block randomly from the netlist, regardless of type.